### PR TITLE
New Netskope API V1 Integration 

### PR DESCRIPTION
**Pull request by RazOmry**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Netskope/Integrations/NetskopeAPIv1/NetskopeAPIv1.py [hunk 1]
```diff
@@ -0,0 +1,941 @@
...
+    """
+
+    hostname = args['hostname']
+    page, skip, limit = get_pagination_arguments(args)
+
+    response = client.list_clients_request(query=f'host_info.hostname eq {hostname}',
+                                           limit=limit,
+                                           skip=skip)
+
+    outputs = sum((client['attributes'].get('users') for client in response['data']), [])
```
Comment (yaakovi) [inline Packs/Netskope/Integrations/NetskopeAPIv1/NetskopeAPIv1.py:718]: Nice use of sum 😎

File: Packs/Netskope/IncidentFields/incidentfield-Netskope_Action.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (tomer-pan) [inline Packs/Netskope/IncidentFields/incidentfield-Netskope_Action.json:1]: @RazOmry Please use the common field "Action" instead creating a new one.

File: Packs/Netskope/IncidentFields/incidentfield-Netskope_Event_Type.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (tomer-pan) [inline Packs/Netskope/IncidentFields/incidentfield-Netskope_Event_Type.json:1]: @RazOmry Please use the common field "Event Type" instead creating a new one.

File: Packs/Netskope/IncidentFields/incidentfield-Netskope_Access_Method.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedTypes": [
+        "Netskope Compromised Credentials",
+        "Netskope Malware",
+        "Netskope Qurantine"
```
Comment (idovandijk) [inline Packs/Netskope/IncidentFields/incidentfield-Netskope_Access_Method.json:23]: Might want to fix typo here - Quarantine

File: Packs/Netskope/IncidentFields/incidentfield-Netskope_Quarantine_File_ID.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedTypes": [
+        "Netskope Qurantine"
```
Comment (idovandijk) [inline Packs/Netskope/IncidentFields/incidentfield-Netskope_Quarantine_File_ID.json:21]: Fix typo - Quarantine

File: Packs/Netskope/IncidentFields/incidentfield-Netskope_Quarantine_Policy.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedTypes": [
+        "Netskope Qurantine"
```
Comment (idovandijk) [inline Packs/Netskope/IncidentFields/incidentfield-Netskope_Quarantine_Policy.json:21]: Same typo

File: Packs/Netskope/IncidentFields/incidentfield-Netskope_Quarantine_Profile.json [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedTypes": [
+        "Netskope Qurantine"
```
Comment (idovandijk) [inline Packs/Netskope/IncidentFields/incidentfield-Netskope_Quarantine_Profile.json:21]: same

File: Packs/Netskope/IncidentFields/incidentfield-Netskope_Quarantine_Profile_ID.json [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedTypes": [
+        "Netskope Qurantine"
```
Comment (idovandijk) [inline Packs/Netskope/IncidentFields/incidentfield-Netskope_Quarantine_Profile_ID.json:21]: same

File: Packs/Netskope/Classifiers/classifier-NetSkope_Classifier.json [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+        "anomaly": "Netskope Alert",
+        "application": "Netskope Event",
+        "audit": "Netskope Event",
+        "infrastructure": "Netskope Event",
+        "malsite": "Netskope Malicious Sites",
+        "malware": "Netskope Malware",
+        "network": "Netskope Event",
+        "page": "Netskope Event",
+        "policy": "Netskope Alert",
+        "quarantine": "Netskope Qurantine",
```
Comment (idovandijk) [inline Packs/Netskope/Classifiers/classifier-NetSkope_Classifier.json:20]: Not sure if intentional - should be Quarantine I assume

File: Packs/Netskope/Classifiers/classifier-NetSkope_Mapper.json [hunk 10]
```diff
@@ -0,0 +1,283 @@
...
+                    "complex": null,
+                    "simple": "object"
+                },
+                "Netskope Malware Type": {
+                    "complex": null,
+                    "simple": "malware_type"
+                }
+            }
+        },
+        "Netskope Qurantine": {
```
Comment (idovandijk) [inline Packs/Netskope/Classifiers/classifier-NetSkope_Mapper.json:202]: same

**Review by idovandijk — CHANGES_REQUESTED** [review]
Not sure if that is a trailing error but there are multiple references to "Qurantine" type instead of "Quarantine", please double check where needed

File: Packs/Netskope/Integrations/NetskopeAPIv1/NetskopeAPIv1.py [hunk 11]
```diff
@@ -0,0 +1,941 @@
...
+
+    for event_type in event_types:
+        new_events = client.list_events_request(start_time=start_time,
+                                                end_time=date_to_seconds_timestamp(datetime.now()),
+                                                limit=max_fetch,
+                                                unsorted=False,
+                                                event_type=event_type)['data']
+        for event in new_events:
+            event['event_id'] = event['_id']
+            event['incident_type'] = event_type
```
[the remark this answers is not in the document]
Comment (RazOmry) [reply Packs/Netskope/Integrations/NetskopeAPIv1/NetskopeAPIv1.py:818]: No. The goal of this loop is to add `event_id` and `incident_type` keys into the existing event. It is not a new object, just modifying the existing one.

**Review by idovandijk — COMMENTED** [review]
From my side this is good to go. @yaakovi if you're also done feel free to approve

File: Packs/Netskope/Integrations/NetskopeAPIv1/README.md [hunk 12]
```diff
@@ -0,0 +1,754 @@
...
+```
+
+#### Human Readable Output
+
+>URL List Allowed URLs:
+>allow.me, allow2.me
+
+### netskope-file-hash-list-update
+***
+Update file hash list with the values provided.
```
Comment (DanilVil) [inline Packs/Netskope/Integrations/NetskopeAPIv1/README.md:492]: @RazOmry We need to state that the update action will rewrite the existing values in the list and doesn't add to them.

File: Packs/Netskope/Integrations/NetskopeAPIv1/README.md [hunk 13]
```diff
@@ -0,0 +1,754 @@
...
+
+
+### netskope-url-list-update
+***
+Update the URL List with the values provided.
+
+
+#### Base Command
+
+`netskope-url-list-update`
```
Comment (DanilVil) [inline Packs/Netskope/Integrations/NetskopeAPIv1/README.md:452]: @RazOmry We need to state that the update action will rewrite the existing values in the list and doesn't add to them.

File: Packs/Netskope/Integrations/NetskopeAPIv1/README.md [hunk 14]
```diff
@@ -0,0 +1,754 @@
...
+    }
+}
+```
+
+#### Human Readable Output
+
+>URL List Allowed URLs:
+>allow.me, allow2.me
+
+### netskope-file-hash-list-update
```
Comment (DanilVil) [inline Packs/Netskope/Integrations/NetskopeAPIv1/README.md:490]: @RazOmry We need to state that the update action will rewrite the existing values in the list and doesn't add to them.

**Review by DanilVil — CHANGES_REQUESTED** [review]
ן

**Pull request merged**